style.css

```diff
@@ -0,0 +1,598 @@
+* {
+  --main-bg-color: #fff;
+  --text-color: #6070ff;
+  --links-color: #344563;
+  --intro-color: #091e42;
+  --primary-B300: #7f8cff;
+  --icon-color: #505f79;
+  --body-color: #e5e5e5;
+  --desc-color: #091e42;
+  --experience: #6b778c;
+  --main-font-family: 'Poppins';
+  --footer-background: #6070ff;
+
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-style: normal;
+  background-color: var(--body-color);
+  font-family: var(--main-font-family);
+}
+
+.main {
+  width: 100%;
+  height: 916px;
+}
+
+.header-1 {
+  background-color: var(--main-bg-color);
+  padding: 25px 155px;
+  flex-direction: row;
+}
+
+nav {
+  display: flex;
+}
+
+.logo {
+  color: var(--text-color);
+}
+
+.my-links {
+  display: flex;
+  padding-left: 75%;
+  gap: 10px;
+  color: var(--links-color);
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+}
+
+ul {
+  list-style: none;
+}
+
+a:link {
+  text-decoration: none;
+}
+
+a:hover {
+  color: var(--text-color);
+}
+
+.intro-part {
+  position: absolute;
+  width: 100%;
+  background-image: url("images/Header\ bg@2x.png");
+  background-size: cover;
+  background-color: var(--main-bg-color);
```

Comment:
Please make sure to add the corresponding background image to all your sections 😉. Kindly notice that this path that you added doesn't exist inside your project's folder, so there isn't any background image at all 😄:
| Figma design  | Your design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/42799579/207967496-c0963689-c1a0-4b1c-a23d-6a0c2ac5c9fe.png)  | ![image](https://user-images.githubusercontent.com/42799579/207967387-208bdcda-1acf-4d2c-86fb-8862700dfe79.png)  |